src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/PaymentAccountValidator.java

```diff
@@ -51,7 +51,8 @@ public static void checkPbaNumberIsValid(Set<String> paymentAccounts) {
                 .collect(Collectors.joining(", "));
 
         if (!StringUtils.isEmpty(invalidPbas)) {
-            throw new InvalidRequest("PBA numbers must start with PBA/pba and be followed by 7 alphanumeric characters. The following PBAs entered are invalid: " + invalidPbas);
+            throw new InvalidRequest("PBA numbers must start with PBA/pba and be followed by 7 alphanumeric "
```

Comment:
concat can be used and also needs refactoring for moving complete string to constant